packages/adapter-node/package.json

```diff
@@ -32,6 +32,7 @@
 		"prepublishOnly": "pnpm build"
 	},
 	"devDependencies": {
+		"@polka/url": "^1.0.0-next.21",
```

Comment:
Doesn't this need to be a regular `dependency` if we're using it in the code the user gets?

Reply:
No, because it gets bundled.